content/spin/spin-storage.md

```diff
@@ -0,0 +1,24 @@
+title = "Persistent Data: Spin"
+template = "spin_main"
+date = "2023-03-14T00:22:56Z"
+enable_shortcodes = true
+
+---
+- [Key/Value Store](#keyvalue-store)
+- [Other Persistent Storage Concepts in Webassembly Applications](#other-persistent-storage-concepts-in-webassembly-applications)
+
+## Key/Value Store
+
+The key/value store option is great for Spin users who would like to store non-relational, data pairs. Spin has a built-in key/value store mechanism that follows BASE consistency principles; a great fit for workloads such as value caching, session caching, counters, and also serializing an application's state.
+
+When you configure your Spin application to use a default key/value store, you will get a 1:1 mapping of your Spin application to a key/value store. The key/value store, built using SQLite, will persist until you delete the directory in which your Spin application was created. SQLite is embedded in a way that is analogous to how one would imagine using a software library i.e. SQLite is designed to be used in this manner and therefore, for all intents and purposes, you could say that SQLite does not exist outside of Spin. When you run your Spin application using the [spin up](https://developer.fermyon.com/common/cli-reference#up) command SQLite is available to your application. When your application is no longer running, neither is SQLite. Your data will continue to persist at all times (including during restarts) and will support the running of your application up until you delete your application directory. 
```

Comment:
> until you delete your application directory

This is not _quite_ true.  You can delete the backing store without deleting the entire app directory.

Reply:
I had a good think about this and I think that it is safe, for now, to just mention the directory only (from a store perspective). Opening up the option of tinkering with the store might raise questions like, "how do I back up my store independently of Spin" and so on. In saying this, these extra more detailed pieces of information may come in handy for advanced users in the future. For now, I think this is fine in terms of a basic use case.